packages/template-retail-react-app/app/pages/checkout/partials/checkout-footer.jsx

```diff
@@ -58,11 +58,11 @@ const CheckoutFooter = ({...otherProps}) => {
 
                     <Box {...styles.bottomHalf}>
                         <Text {...styles.copyright}>
-                            &copy;{' '}
+                            &copy; {new Date().getFullYear()}{' '}
                             {intl.formatMessage({
                                 id: 'checkout_footer.message.copyright',
                                 defaultMessage:
-                                    '2021 Salesforce or its affiliates. All rights reserved. This is a demo store only. Orders made WILL NOT be processed.'
+                                    'Salesforce or its affiliates. All rights reserved. This is a demo store only. Orders made WILL NOT be processed.'
```

Comment:
Can you repeat the change for the other footer file too? Thanks.

https://github.com/bfeister/pwa-kit/blob/7cb911ed673164cb7c9389448bf5844e7e3b1ab7/packages/template-retail-react-app/app/components/footer/index.jsx#L166-L173

And yeah, unfortunately doing so would need another round of translation updates, because they use a different message id `footer.message.copyright`.

Reply:
Done! 👍